src/test/ui/error-codes/E0603.stderr

```diff
@@ -4,7 +4,7 @@ error[E0603]: constant `PRIVATE` is private
 LL |     SomeModule::PRIVATE;
    |                 ^^^^^^^ this constant is private
    |
-note: the constant `PRIVATE` is defined here
+note: constant `PRIVATE` defined here
```

Comment:
@varkor 
Sorry to bother you again, but this is not a label, primary messages use full sentences.

Reply:
Isn't `lint level defined here` a primary message, yet that does not use a full sentence?

Reply:
Well, there are inconsistencies in existing diagnostics.
I wouldn't personally bother changing them especially if it's causing churn, I just want to avoid new primary messages using nominal forms and label messages being wordy.

Reply:
We could document this guidance in rustc guide, but it won't help because few people read it.
For lints we have a whole official accepted RFC about naming which nobody remembers.

Reply:
Fixing the inconsistencies was the entire point of this PR. I'll try to make them conform to the style you suggest.

Reply:
After some thought, perhaps it's better to document it in rustc-guide after all, even if few people notice it by just reading the guide, it's still some written text to link people to.